src/ViewLayer/RaceModeDisplay/RaceModeDashboardUI.ui.orig

```diff
@@ -0,0 +1,3956 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file being used anymore?

Reply:
.orig files are artifacts from git merges, shouldn't keep them.
Remove and add to the .gitignore!